test/usage.test.ts

```diff
@@ -299,6 +303,28 @@ describe("feedback prompting", () => {
     expect(closeAblySSEChannel).toBeCalledWith("fake_client");
   });
 
+  test("does not call tracking endpoint if token cached", async () => {
```

Comment:
I can't make sense of the title of the test. It doesn't seem like we are testing that anything is specifically not called. There is no usage of nock for request interception/assertion and there is no assertion to a spy not being called. Is there something missing here?

Reply:
This would fail exactly because there is no nock expecting a call. Also the ``` vi.mocked(getAuthToken).mockReturnValue({
      channel: "super-channel",
      token: "something",
    });
``` guarantees that `token`'s value `something` can only come from the storage as there is no other place that can get you a token.

Reply:
We're actually not setting up nock anywhere in this test file, except for in individual other tests. As far as I can see, there is no nock instance active which could detect an unwanted request. If there is actually one, then it must be leaked from another test, which would be problematic.

Reply:
I probably explained incorrectly -- this test would fail if a nock is installed or not, if the code tries to call the API to obtain an auth token. So the test effectively tests that no such call is ever made. I can include a nock to explicitly show that.

Reply:
@Munter, done, added an explicit check